docs/tutorials/native-vrf/commit-reveal-cadence.md

```diff
@@ -48,25 +49,28 @@ You'll need the following:
 
 In this guide, we will explore how to use a commit-reveal scheme in conjunction with Flow's Random Beacon to achieve secure, non-revertible randomness. This mechanism mitigates post-selection attacks, where participants attempt to manipulate or reject unfavorable random outcomes after they are revealed.
 
-To illustrate this concept, we will build a Coin Toss game on Flow, demonstrating how smart contracts can leverage commit-reveal randomness for fair, tamper-resistant results.
+To illustrate this concept, we will build a Coin Toss game on Flow, demonstrating how smart contracts can leverage a commit-reveal scheme for fair, tamper-resistant results.
 
 ### What is the Coin Toss Game?
 
-The Coin Toss Game is a decentralized betting game that showcases Flow's commit-reveal randomness. Players place bets without knowing the random outcome, ensuring fairness and resistance to manipulation.
+The Coin Toss Game is a decentralized betting game that showcases Flow's commit-reveal pattern. Players place bets without knowing the random outcome, ensuring fairness and resistance to manipulation.
 
 The game consists of two distinct phases:
 
-1. Commit Phase – The player places a bet by sending Flow tokens to the contract. The contract records the commitment and requests a random value from Flow's Random Beacon. The player receives a Receipt, which they will use to reveal the result later.
-2. Reveal Phase – Once the random value becomes available in `RandomBeaconHistory`, the player submits their Receipt to determine the outcome:
+1. Commit Phase – The player places a bet by sending Flow tokens to the contract. The contract records the commitment to use a future random value from Flow's Random Beacon. The player receives a Receipt, which they will use to reveal the result later.
+2. Reveal Phase – Once the random value becomes available in the `RandomBeaconHistory` contract, the player submits their Receipt to determine the outcome:
    - If the result is 0, the player wins and receives double their bet.
    - If the result is 1, the player loses, and their bet remains in the contract.
 
-### Why Use Commit-Reveal Randomness?
+### Why Use a Commit-Reveal scheme?
 
-- Prevents manipulation – Players cannot selectively reveal results after seeing the randomness.
+Similarly to revertible randomness, Commit-Reveal inherits the security of Flow native randomness beacon: 
```

Comment:
separating the advantages of Flow randomness (revertible or not), and then commit-reveal (compared to revertible)